sparse/coo/umath.py

```diff
@@ -481,8 +481,8 @@ def _get_fill_value(self):
             fill_value = self.func(*zero_args, **self.kwargs)[()]
 
         if not equivalent(fill_value, fill_value_array).all():
-            raise ValueError('Inconsistent fill-values in the result array: operating on the ndarray with'
-                             ' fill-values produces inconsistent results.')
+            raise ValueError('Performing a mixed sparse-dense operation that would result in a dense array. '
+                             'Please make sure that func(sparse_fill_values, ndarrays) is a constant array.')
```

Comment:
What is func?

Reply:
The element-wise function one wants to apply.